python/fio/test_fio_plugin.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import unittest
+import json
+from pathlib import Path
+
+import yaml
+from arcaflow_plugin_sdk import plugin
+
+import fio_plugin
+import fio_schema
+
+
+with open("mocks/poisson-rate-submission_output-plus.json", "r") as fout:
```

Comment:
You have both `fixtures` and `mocks` subdirs in the repo with separate references in the README and the code.